test/sql/test_metrics/R/test_memory_metrics

```diff
@@ -0,0 +1,102 @@
+-- name: test_memory_metrics_running_task_run_count
+shell: curl '${url}/metrics' | grep starrocks_fe_memory | grep running_task_run_count | wc -l
+-- result:
+0
+1
+-- !result
+-- name: test_memory_metrics_compaction_stats_count
+shell: curl '${url}/metrics' | grep starrocks_fe_memory | grep compaction_stats_count | wc -l
+-- result:
+0
+1
+-- !result
+-- name: test_memory_metrics_txn_callback_count
+shell: curl '${url}/metrics' | grep starrocks_fe_memory | grep txn_callback_count | wc -l
+-- result:
+0
+1
+-- !result
+-- name: test_memory_metrics_catalogs_count
+shell: curl '${url}/metrics' | grep starrocks_fe_memory | grep catalogs_count | wc -l
+-- result:
+0
+1
+-- !result
+-- name: test_memory_metrics_stream_load_task_count
+shell: curl '${url}/metrics' | grep starrocks_fe_memory | grep stream_load_task_count | wc -l
+-- result:
+0
+1
+-- !result
+-- name: test_memory_metrics_txn_bytes
+shell: curl '${url}/metrics' | grep starrocks_fe_memory | grep txn_bytes | wc -l
+-- result:
+0
+1
+-- !result
+-- name: test_memory_metrics_task_count
+shell: curl '${url}/metrics' | grep starrocks_fe_memory | grep task_count | wc -l
+-- result:
+0
+2
+-- !result
+-- name: test_memory_metrics_txn_count
+shell: curl '${url}/metrics' | grep starrocks_fe_memory | grep txn_count | wc -l
+-- result:
+0
+1
+-- !result
+-- name: test_memory_metrics_pending_task_run_count
+shell: curl '${url}/metrics' | grep starrocks_fe_memory | grep pending_task_run_count | wc -l
+-- result:
+0
+1
+-- !result
+-- name: test_memory_metrics_replica_bytes
+shell: curl '${url}/metrics' | grep starrocks_fe_memory | grep replica_bytes | wc -l
+-- result:
+0
+1
+-- !result
+-- name: test_memory_metrics_tablet_bytes
+shell: curl '${url}/metrics' | grep starrocks_fe_memory | grep tablet_bytes | wc -l
```

Comment:
should we create some tables to check if these metrics are correct?

Reply:
The cluster can create many tables, it's not easy to calculate the number.

Reply:
we don't need a precise number, just  check `> xxx`

Reply:
does sql test support > xxx ?